timeparser_test.go

```diff
@@ -80,7 +80,16 @@ func Test_ParseLayout4_2(t *testing.T) {
 
 func Test_ParseLayout5(t *testing.T) {
 	date, err := parseTime("22 Jul 2013 14:55:01 EST")
-	expected := time.Date(2013, time.July, 22, 14, 55, 1, 0, time.FixedZone("EST", -18000))
```

Comment:
why has this been changed from -18000 to 0?

Reply:
this test case was failing before. 

EST already has offset. So there should not be offset if timezone is already stated as EST